pkg/scheduler/framework/plugins/defaultassigner/algorithm.go

```diff
@@ -47,7 +47,7 @@ func DynamicDivideReplicas(sub *appsapi.Subscription, deviations []appsapi.FeedO
 		if err != nil {
 			return *result, err
 		}
-		result.Merge(&r)
+		result.MergeOneFeed(&r)
```

Comment:
This is more like a renaming.
It seems old method `Merge` will be never used.

Reply:
We can not guarantee that there will be only one feed.

Reply:
MergeOneFeed is a special case of Merge.
Maybe in the future we need Merge in another plugin.
Now comment it?

Reply:
> We can not guarantee that there will be only one feed.

Exactly. Maybe keep the original `Merge` function would be better even if it will not be used now after this simplification.

Reply:
Keeping `Merge` in the historical version is also possible. If we need it, we can always add it back.
[here](https://github.com/clusternet/clusternet/blob/v0.14.0/pkg/scheduler/framework/interfaces/types.go#L100)